internal/impl/babysitter.go

```diff
@@ -211,6 +228,10 @@ func (b *babysitter) watchPods(ctx context.Context, component string) error {
 					delete(addrs, pod.Name)
 					changed = true
 				}
+			case watch.Error:
+				errObject := apierrors.FromObject(event.Object)
+				logger.Error("watcher error", "err", errObject)
```

Comment:
can we inline the errObject? 

`b.logger.Error("watcher error", "err", apierrors.FromObject(event.Object))`

Reply:
done